spec/spec_helper.rb

```diff
@@ -11,6 +11,7 @@
 
 require "webmock/rspec"
 require "timecop"
+require "helpers/helpers"
```

Comment:
You do not need this line if you move the file to `spec/support` which is where it should live :)

Reply:
Done in sha: 5e2fd92